extensions/wikia/Flags/models/FlagsModel.class.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Flags\Models;
+
+class FlagsModel extends \WikiaModel {
+
+	const FLAGS_TO_PAGES_TABLE = 'flags_to_pages';
+	const FLAGS_TYPES_TABLE = 'flags_types';
+	const FLAGS_PARAMS_TABLE = 'flags_params';
+
+	protected function getDatabaseForRead() {
```

Comment:
I'm not sure of the value of these over the method WikiaModel already has:
`$this->getSharedDB();`
`$this->getSharedDB( DB_MASTER );`